componentLibraries/extensionLibrary/extensionLibrary.xml

```diff
@@ -18,6 +18,8 @@
     <cflags>-I./someLib/include</cflags>
     <lflags>-L./someLib/bin -lsomeLib</lflags>
     <lflags os="win64">-lWs2_32</lflags> -->
+	<cflags>-I../../dependencies/fmilibrary/include</cflags>
```

Comment:
Hmm a bit non-portable. But I guess there is no easy fix for that. We would need to know the HopsanInstallation directory automatically somehow.

Reply:
We should use the $HOPSANHOME environment variable in such cases I think.  But this will do for now.